corehq/util/email_event_utils.py

```diff
@@ -12,6 +14,18 @@
 )
 
 
+def log_email_sns_event(message):
+    log_event = {
+        'eventType': message.get('eventType'),
+        'eventTimestamp': message.get('mail', {}).get('timestamp'),
+        'commonHeaders': message.get('mail', {}).get('commonHeaders')
+    }
+    for key in ['bounce', 'complaint', 'delivery', 'reject', 'failure', 'deliveryDelay']:
+        if key in message:
+            log_event[key] = message.get(key)
+    logging.info(log_event)
```

Comment:
Looks like our default logging settings already report anything with log-level info or above? Or did you have to do something specifically to get this to show up in the file?

Reply:
I was originally using the [getLogger](https://github.com/dimagi/commcare-hq/pull/28522/commits/3bd6d521dd8d9490c577fe6c0ac4a9306c2790a5#diff-7a0fac04d9f436fa54bb541709e8177aR16) so that was why it wasn't working?